Project2/style.css

```
@@ -0,0 +1,115 @@
+body {
+    box-sizing: border-box;
+    height: auto;
+    width: 640px;
+}
+
+.mainimage {
+    background-image: url(Images/TheFavouriteWebCardImage.jpg);
+    background-size: cover;
+    height: 380px;
+    position: relative;
+    width: 640px;
+    z-index: 3;
+
+
+}
+
+
+
+.content {
+    background: linear-gradient(to left, #42513F, transparent 190%);
+    height: 100%;
+    width: 640px;
+}
+
+
+.details {
+    left: 410px;
+    padding: 10px;
+    position: relative;
+    top: 1px;
+    width: 200px;
+
+
+}
+
+
+
+
+h1 {
+    bottom: -20px;
+    color: #DB5756;
+    font-size: 20pt;
```

For browsers, you want to provide your `font-size` value in pixels:
```suggestion
    font-size: 20px;
```
There are some alternatives, but for these projects, I'm asking y'all to use pixels exclusively. Note that 20px is _not_ the same as 20pt, so you'd probably want to increase this value, too (28px is probably close to what size you have now).